new.csv

```diff
@@ -0,0 +1,5 @@
+month,avg,max,min
```

Comment:
what is this file? i don't think it should be here

Reply:
removed now